addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -258,6 +258,128 @@ QUnit.test('chat window new message: fold', async function (assert) {
     );
 });
 
+QUnit.test('chat window new message: replace "new message" window by "opened chat" window', async function (assert) {
```

Comment:
Would omit _"chat window new message:"_ in naming (already clear with rest of name, so redundant).

Would even name test like an assertion comment, e.g. _"open chat from 'new message' chat window should open chat in place of this 'new message' chat window"_.

Reply:
ok